profiles/lib/way_handlers.lua

```diff
@@ -521,7 +521,7 @@ function WayHandlers.blocked_ways(profile,way,result,data)
     local construction = way:get_value_by_key('construction')
 
     -- Of course there are negative tags to handle, too
-    if construction and construction ~= 'no' and construction ~= 'widening' then
+    if construction and construction ~= 'no' and construction ~= 'widening' and construction ~= 'minor' then
```

Comment:
We could check for `https://github.com/Project-OSRM/osrm-backend/blob/master/profiles/car.lua#L143-L156` or some similar list, right? That would avoid hardcoding more strings there.